lib/constable_web/controllers/api/announcement_controller.ex

```diff
@@ -8,8 +8,44 @@ defmodule ConstableWeb.Api.AnnouncementController do
 
   plug(Constable.Plugs.Deslugifier, slugified_key: "id")
 
-  def index(conn, _params) do
-    announcements = Repo.all(Announcement)
+  def index(conn, params = %{"after" => after_id}) do
```

Comment:
What do you think about adding some tests for each of these scenarios? I'm not sure what the state of the api tests are, but maybe a good idea?

Reply:
I think @danbee mentioned he was planning to add some tests.